proxy/src/main/java/org/dragonet/proxy/network/translator/pe/PEInteractPacketTranslator.java

```diff
@@ -17,23 +17,35 @@
 import org.dragonet.proxy.network.UpstreamSession;
 import org.dragonet.proxy.network.translator.IPEPacketTranslator;
 import com.github.steveice10.mc.protocol.packet.ingame.client.player.ClientPlayerInteractEntityPacket;
+import org.dragonet.proxy.network.cache.CachedEntity;
 import org.dragonet.proxy.protocol.packets.InteractPacket;
 
 public class PEInteractPacketTranslator implements IPEPacketTranslator<InteractPacket> {
-	// vars
+    // vars
 
-	// constructor
-	public PEInteractPacketTranslator() {
+    // constructor
+    public PEInteractPacketTranslator() {
 
-	}
+    }
 
-	// public
-	public Packet[] translate(UpstreamSession session, InteractPacket packet) {
-		ClientPlayerInteractEntityPacket pk = new ClientPlayerInteractEntityPacket(
-				(int) (packet.targetRtid & 0xFFFFFFFF), InteractAction.ATTACK);
-		return new Packet[] { pk };
-	}
+    // public
+    public Packet[] translate(UpstreamSession session, InteractPacket packet) {
+        CachedEntity e = session.getEntityCache().getByLocalEID(packet.targetRtid);
```

Comment:
Finally someone who knows how to use the TAB button!

Reply:
For explanations : the entire project is coded with `\t`, wich is horrible.
I'm replacing all by four space tabs when I hardly work on a class.
I'll do the conversion on all classes i pass now.